pages/en/lb3/readmes/loopback-connector-grpc.md

```diff
@@ -0,0 +1,125 @@
+# loopback-connector-grpc
```

Comment:
I think this file is copied using the script https://github.com/strongloop/loopback.io/blob/gh-pages/update-readmes.sh.

Reply:
That's how I created the two files, but I was looking at the `gh-pages` branch and the other connectors' readmes appear [here](https://github.com/strongloop/loopback.io/tree/gh-pages/pages/en/lb3/readmes) so that's why I included them. If I remove them, will they be generated when it's deployed and added to the `gh-pages` branch?

Reply:
@nabdelgadir are those two files created by some command or you manually added them here? If they are auto generated I guess you are doing the right thing, according to https://github.com/strongloop/loopback.io#incorporating-external-readmes :)

Reply:
_I think_ it will get copied automatically in a travis job. See https://github.com/strongloop/loopback.io/commit/d9fab70b305cf3c5596e76e1bf422e6c20be8eb9. But I'm good to commit the changes here just in case.

Reply:
@jannyHou they were autogenerated by `sh update-readmes.sh`(so I could see the pages when running locally).

@dhmlau ohh okay, I'll keep them then. 👍